packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -79,6 +79,13 @@ export class CdkToolkit {
     return stacks.firstStack.manifest.metadata ?? {};
   }
 
+  public async acknowledge(noticeId: string) {
```

Comment:
The idea here is that the scope of acknowledgement should be on a per-application basis. So I'm storing the acknowledgements as context in `cdk.context.json`.